autoload/lsc/edit.vim

```diff
@@ -81,6 +81,10 @@ function! lsc#edit#rename(...) abort
   else
     let new_name = input('Enter a new name: ')
   endif
+  if (empty(trim(new_name)))
+    echo 'Name can not be blank'
```

Comment:
Can you use `lsc#message#error` here?

Reply:
Thank you, I didn't know it.

Reply:
I'm confused by the `echo "\n"` What is that accomplishing? Does the error message otherwise not show up?

Reply:
Because other wise, error message would show on the same line as the input prompt
![Screen Shot 2019-03-15 at 11 53 50](https://user-images.githubusercontent.com/1186411/54405173-1abf2600-4719-11e9-84d9-42f1992070f9.png)

Reply:
Ah makes sense.